### ISSUE TITLE
Document delay

### ISSUE THREAD

**Issue by wbt** [tablelandnetwork/js-tableland#85]
It would be useful to have documentation (especially around read/SELECT functions) discussing the delay between when an update transaction succeeds and when the change is likely to be reflected in the results of a table read / SELECT statement call.  There are plenty of use cases where someone might take an action that appears to succeed, and then immediately reload a view expecting it to reflect the updated state.  If that view gets data from a read/SELECT query, it won't reflect the changed data right away, leading the user to think the change was not effective, and possibly taking additional actions such as trying to repeat the state change, which might make things worse.   There's currently no documentation about the source(s) or extent of this delay or if a developer can do anything to tell if it's over.

**Comment by joewagner** [comment]
Hey @wbt
**TL;DR;**  The functionality you're requesting is already working on the `main` branch, and will be included in the next release.

**More detail**
We have been using the term "materialization" to describe how the validator listens to events and turns them into a SQLite database.  Basically the chain acts as the source of truth and all of the Validators in our network then materialize the state of event data when transactions are included in blocks.
There is an RPC endpoint named `tableland_getReceipt` that can be used to check if a given transaction hash has been materialized by a given Validator.
As of today, there is only one Validator (run by the core team) so this isn't that complicated.  However in the near future there will be many Validators, and in preparation for that we are building out documentation of all of this.  At the moment it's not quite finished though.

**Comment by joewagner** [comment]
I closed this, but feel free to reopen if you don't feel you issue is resolved
